warehouse/query-core/src/main/java/datawave/query/iterator/QueryIterator.java

```diff
@@ -1328,7 +1329,8 @@ protected IteratorBuildingVisitor createIteratorBuildingVisitor(Class<? extends
                         .setIvaratorCacheBufferSize(this.getIvaratorCacheBufferSize())
                         .setIvaratorCacheScanPersistThreshold(this.getIvaratorCacheScanPersistThreshold())
                         .setIvaratorCacheScanTimeout(this.getIvaratorCacheScanTimeout()).setMaxRangeSplit(this.getMaxIndexRangeSplit())
-                        .setIvaratorMaxOpenFiles(this.getIvaratorMaxOpenFiles()).setIvaratorSources(this, this.getMaxIvaratorSources())
+                        .setIvaratorMaxOpenFiles(this.getIvaratorMaxOpenFiles()).setIvaratorSource(this.sourceForDeepCopies)
```

Comment:
maybe setIvaratorSource should be setUnsortedIvaratorSource

Reply:
sure.  done.